TesseractCore/Exporting.swift

```diff
@@ -1,7 +1,7 @@
 //  Copyright (c) 2015 Rob Rix. All rights reserved.
 
 public func export<T>(graph: Graph<T>) -> String {
-	let edges = join("\n", lazy(graph.edges).map { "\t" + $0.source.identifier.description + " -> " + $0.destination.identifier.description + ";" })
+	let edges = join("\n", lazy(graph.edges).map { "\t\"" + $0.source.identifier.description + "\" -> \"" + $0.destination.identifier.description + "\";" })
```

Comment:
Identifiers are likely to change (or even be parameterizable) eventually, so this is a very good call :+1: